### DISCUSSION TITLE
Cables on Wordpress not working after migration

### DISCUSSION THREAD

**Discussion by helmiaddibani** [cables-gl/cables_docs#190] [Answers & Questions] [answered]
Hey all,

I was just migrating my website on new hosting. I'm using cables plugin to embed the cables patch on my wordpress website. I was using https://x10hosting.com/ (a free hosting) to rainbow-web.com for the new hosting for the live version. Unfortunately the page is not even opening where there is cables patch and giving an error message (see bellow). Is anyone having similar issue? Can anyone assist me to solve the problem? Thanks in advance
![Screen Shot 2022-03-07 at 19 10 41](https://user-images.githubusercontent.com/13184243/157032653-a1650109-7320-4ebe-98f2-324fc2b1f636.png)

**Comment by steam0r** [comment]
could you contact me on discord, would be interesting to see server-logfiles as well, if possible. discord usersname is "steam"

**Comment by steam0r** [comment, marked as answer]
for everyone reading this in the future:

it was a problem with how the hoster handles "php shortopentag" and is fixed in version 0.12.3 of the plugin
